src/ts_port.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2021 Bobby Noelte.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * ThingSet communication port handling
+ */
+
+#ifndef TS_PORT_H_
+#define TS_PORT_H_
+
+#include <stdint.h>
+
+#include "ts_time.h"
+#include "ts_buf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief ThingSet node identifier.
+ *
+ * A node identifier identifies a specific node that can be accessed by a port. The translation
+ * of the node identifier toport specific adressing (e.g. CAN ID) has to be done by the port
+ * implementation.
```

Comment:
The node ID could also be equivalent to the device ID (currently 8 character string in Libre Solar devices), correct?

Added one sentence at the bottom and fixed two typos:

```suggestion
 * A node identifier identifies a specific node that can be accessed by a port. The translation
 * of the node identifier to port specific addressing (e.g. CAN ID) has to be done by the port
 * implementation.
 *
 * The node identifier may be equivalent to a device ID as exposed through the application
 * layer protocol.
```

Reply:
Changed